openreview/conference/templates/programchairWebfield.js

```diff
@@ -971,6 +984,7 @@ var displaySortPanel = function(container, sortOptions, sortResults, searchResul
   $(container).html(
     '<form class="form-inline search-form clearfix" role="search">' +
       '<div class="pull-left"></div>' +
+      '<div class="pull-middle"></div>' +
```

Comment:
I think it would look better to have the button right next to the other button in the left-aligned section.

Reply:
Do you mean inside of `pull-left` div? like this?

```
    $(container).find('form.search-form .pull-left').html('<div class="btn-group message-papers-container" role="group">' +
      '<button type="button" class="message-papers-btn btn btn-icon dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="Select papers to message corresponding reviewers" disabled="disabled">' +
        '<span class="glyphicon glyphicon-envelope"></span> &nbsp;Message Reviewers ' +
        '<span class="caret"></span>' +
      '</button>' +
      '<ul class="dropdown-menu">' +
        '<li><a class="msg-all-reviewers">All Reviewers of selected papers</a></li>' +
        '<li><a class="msg-submitted-reviewers">Reviewers of selected papers with submitted reviews</a></li>' +
        '<li><a class="msg-unsubmitted-reviewers">Reviewers of selected papers with unsubmitted reviews</a></li>' +
      '</ul>' +
    '</div>' +
    '<div><button class="btn btn-export-data">Export</button></div>');
    renderTable(container, rowData);
```